src/main/java/core/basesyntax/impl/FruitStoreServiceImpl.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.FruitStoreService;
+import core.basesyntax.strategy.ManipulationService;
+import core.basesyntax.strategy.Manipulations;
+import java.util.List;
+import java.util.Map;
+
+public class FruitStoreServiceImpl implements FruitStoreService {
+    private static final int INDEX_OF_TYPE = 0;
+    private static final int INDEX_OF_FRUIT_NAME = 1;
+    private static final int INDEX_OF_QUANTITY = 2;
+
+    public List<Fruit> changeBalance(List<String> text, Map<Manipulations, ManipulationService>
+            typeActivityMap, FruitStorageDao storageDao) {

```

Comment:
dependencies (services on which your service depends) should be passed to constructor, not to method params